qutebrowser/config/configdata.yml

```diff
@@ -3315,7 +3315,7 @@ colors.webpage.darkmode.policy.page:
   restart: true
   backend: QtWebEngine
 
-colors.webpage.darkmode.threshold.text:
+colors.webpage.darkmode.threshold.foreground:
```

Comment:
This needs an entry with the old name and a `renamed:` for automatic config migration (you'll find some other examples in the file).

Reply:
Didn't know about that one. Thanks for pointing it out!